Cargo.toml

```diff
@@ -30,6 +30,7 @@ chrono = "0.4"
 error-chain = { git = "https://github.com/rnewman/error-chain", branch = "rnewman/sync" }
 lazy_static = "0.2"
 time = "0.1"
+uuid = "0.5"
```

Comment:
I feel like syncing should be a feature, default-on (take a look at how we configure rusqlite's "bundled" feature). Follow-up is fine.